language_scores.json

```diff
@@ -0,0 +1,312 @@
+{
```

Comment:
Since we can generate this on the fly, no need to include it. 

Also, when generating website, make sure it’s generated in the folder that will be public.

Reply:
Writing the JSON file to `website/dist` now in Github workflow